charts/atlantis/templates/statefulset.yaml

```diff
@@ -3,13 +3,18 @@ kind: StatefulSet
 metadata:
   name: {{ template "atlantis.fullname" . }}
   labels:
-{{- include "atlantis.labels" . | nindent 4 }}
-{{- if .Values.statefulSet.labels }}
-{{ toYaml .Values.statefulSet.labels | indent 4 }}
-{{- end }}
-  {{- with .Values.statefulSet.annotations }}
+    {{- include "atlantis.labels" . | nindent 4 }}
+    {{ with .Values.statefulSet.labels }}
+    {{- toYaml .Values.statefulSet.labels | nindent 4 }}
```

Comment:
This is broken, should be `toYaml .` instead.

Reply:
That's correct. Can you send a fix and we'll merge it pls?

Reply:
And `{{ with` should be `{{- with` to prevent the extra line too.

Reply:
Fix in #364